utils/src/test/kotlin/RedirectionTest.kt

```diff
@@ -58,8 +58,8 @@ class RedirectionTest : WordSpec({
             val stdout = redirectStdout {
                 stderr = redirectStderr {
                     for (i in 1..numberOfLines) {
-                        System.out.println("stdout: $i")
-                        System.err.println("stderr: $i")
+                        println("stdout: $i")
+                        println("stderr: $i")
```

Comment:
This needs to continue to print to `stderr`.

Reply:
Ups, good catch.

Reply:
I removed the commit.